meson.build

```diff
@@ -34,7 +34,6 @@ add_project_arguments('-Os', '-Wall', '-Werror',
                       language : 'c')
 
 glib = dependency('glib-2.0')
-libdl = cc.find_library('dl')
```

Comment:
@giuseppe the dependency has been added as part of #190. Do you know for which environment we required explicitly setting this?